rest-api/src/main/java/org/jboss/pnc/rest/api/endpoints/BuildConfigurationEndpoint.java

```diff
@@ -123,6 +122,7 @@
     })
     @GET
     @Path("/{id}")
+    @Consumes(MediaType.APPLICATION_JSON_PATCH_JSON) //workaround for PATCH support
```

Comment:
What is this doing? And why there is `@Consumes` on a getter method?

Reply:
RestEasy creates a full entity from the JsonPatch request, it does it by internally invoking the GET method mapped on the same Path. The additional media type is required to workaround a RestEasy bug.